docs/source/develop/api/commands.rst

```diff
@@ -21,6 +21,7 @@ Schema
     message AddAssetQuantity {
         string asset_id = 1;
         string amount = 2;
+        optional string title = 3;
     }
```

Comment:
I suggest to use: `.. literalinclude:: ../../../../shared_model/schema/commands.proto`, similar usage You have in file `queries.rst`. But this is not what You must change - this is up to You.